packages/perseus/src/widgets/interactive-graphs/graphs/components/movable-line.tsx

```diff
@@ -150,9 +188,16 @@ const Line = (props: LineProps) => {
             <g
                 ref={line}
                 tabIndex={disableKeyboardInteraction ? -1 : 0}
+                aria-label={ariaLabel}
+                aria-describedby={ariaDescribedBy}
+                aria-live={ariaLive}
                 className="movable-line"
                 data-testid="movable-line"
                 style={{cursor: dragging ? "grabbing" : "grab"}}
+                // Indicate that this element is interactive.
+                // As a bonus, giving this group a non-group role makes
+                // the screen reader skip over its empty children.
+                role="button"
```

Comment:
I re-read the whole graph section of the SRUX doc earlier and noticed I missed this 

> An appropriate semantic element or role for the graph as a whole is [figure](https://khanacademy.atlassian.net/wiki/spaces/LC/pages/3261071420/Proposal+Screen+Reader+User+Experience+SRUX#Semantics).

Reply:
💀 I missed that too.

I think this line is still `button` since it's referring to the grab handle, but I think I'll have to add `figure` to the parent layer.